lib/registries/addon/overview/files/template.hbs

```diff
@@ -1,14 +1,5 @@
 {{page-title (t 'registries.overview.files.title')}}
-<h3>
-    {{!-- swap with file provider name --}}
-    {{t 'registries.overview.files.title'}}
-</h3>
 
 {{#unless this.model.taskInstance.isRunning}}
```

Comment:
Would this make more sense being in `lib/registries/addon/overview/files/provider/template.hbs`?